### PR TITLE
Adding config for deploying the learner MFE for xPro

### PR DESCRIPTION
**Pull request by blarghmatey**
#### What's this PR do?
Adds configuration for deploying the edX learning microfrontend to xPro

### CODE REVIEW

File: pillar/edx/ansible_vars/xpro.sls [hunk 1]
```diff
@@ -113,3 +115,42 @@ edx:
     EDXAPP_BUGS_EMAIL: {{ support_email }}
     EDXAPP_CONTACT_EMAIL: {{ support_email }}
     EDXAPP_TECH_SUPPORT_EMAIL: {{ support_email }}
+
+    # Configuration for managing micro frontends
+    EDXAPP_MANAGE_MICROFRONTENDS: True
+    MFE_DEPLOY_VERSION: open-release/koa.master
+    MFE_DEPLOY_SITENAME: 'xPro'
```
Comment (shaidar) [inline pillar/edx/ansible_vars/xpro.sls:122]: Shouldn't this be `xPRO`?

Comment (blarghmatey) [reply pillar/edx/ansible_vars/xpro.sls:122]: Yeah, guess you're probably right. Changing...

Comment (shaidar) [inline pillar/edx/ansible_vars/xpro.sls:122]: This one didn't change.

**Comment by shaidar** [comment]
👍

**Pull request merged**